tests/test_demes.py

```diff
@@ -1018,6 +1036,44 @@ def test_census_event_is_ignored(self):
         demog.add_census(100)
         demog.to_demes()
 
+    @pytest.mark.filterwarnings(
+        # demes warns about multiple pulses with the same time
+        "ignore:Multiple pulses:UserWarning"
+    )
+    def test_multipulse_roundtrip(self):
```

Comment:
This is really two tests, isn't it? Can just call them ``test_multipulse_roundtrip_ex1``, ``test_multipulse_roundtrip_ex2``